array-methods.js

```diff
@@ -1,14 +1,25 @@
-var dataset = require('./dataset.json');
+var dataset = require("./dataset.json");
 
 /*
   create an array with accounts from bankBalances that are
   greater than 100000
   assign the resulting new array to `hundredThousandairs`
 */
 var hundredThousandairs = null;
+hundredThousandairs = dataset.bankBalances.filter(antiPoverty);
```

Comment:
You don't need to reassign `hundredThousandairs`. You can remove the null and assign the `filter` function directly. Same with the other exercises below.